test/sql/updates/post.catalog.sql

```diff
@@ -6,12 +6,14 @@ SELECT NOT (extversion >= '2.19.0' AND extversion <= '2.20.3') AS has_fixed_comp
   FROM pg_extension
  WHERE extname = 'timescaledb' \gset
 
+\if ! :PG_UPGRADE_TEST
```

Comment:
psql:test/sql/updates/post.catalog.sql:9: error: unrecognized value "! false" for "\if expression": Boolean expected

This doesn't seem to work as expected. Potential fix: https://github.com/timescale/timescaledb/pull/8946